pettingzoo/classic/chess/chess_utils.py

```diff
@@ -321,17 +324,18 @@ def get_observation(orig_board, player):
       }
     """
     # from 0-63
-    square = board.ep_square
+    # Adjust the row number for the white pawn to the 8th if the en passant flag is set, and vice versa for black pawns.
+    square = board.ep_square  # (int) where the en passant happened
     if square:
-        ours = square > 32
+        ours = square < 32
         row = square % 8
-        dest_col_add = 8 * 7 if ours else 0
+        dest_col_add = 0 if ours else 8 * 7
         dest_square = dest_col_add + row
         if ours:
-            result[base + 0].remove(square - 8)
+            result[base + 0].remove(square + 8)
             result[base + 0].add(dest_square)
         else:
-            result[base + 6].remove(square + 8)
+            result[base + 6].remove(square - 8)
```

Comment:
Not strictly necessary but personally I'm not 100% sure what this part of the code does so if you wanted to add a little in line comment for the if and the else part saying what the two scenarios are, that might be helpful

Reply:
I add an example and a few notes to explain how this section works if en passant happened.